### ISSUE TITLE
Steam profile: can't sync or delete

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1033]
The site has been having trouble syncing my steam profile for over a month, it seems. The waitlist is still in the same state it was back then, despite my steam wishlist and steam collection having been changed. Clicking the "sync now" buttons shows a spinner for a second, then "updated now", but refreshing the page makes them show "updated 48 days ago" again. Clicking the buttons does send a POST to the backend, so that part works. However, I also can't delete my steam link from the site. Clicking the delete button triggers a confirmation popup, but at no point does the site send an http request to the backend to actually perform the unlinking.

TLDR: steam syncing is broken, and the website needs a fix to actually send a request to the backend to delete a steam profile link.

[Issue created by rtbf*gsIqJAkoTu6JDdT0yjdimw: 2018-05-28]

**Comment by MarkitkaM** [comment]
I didn't find this profile on Steam "rtbf*gsIqJAkoTu6JDdT0yjdimw"
